CodeStarterCamp_Week5/auditionManager.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  auditionManager.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by MARY on 2023/03/20.
+//
+
+import Foundation
+
+extension Talent {
+    var hasOutstandingTalent: Bool {
+        get {
+            self.singing == Level.A || self.dancing == Level.A || self.acting == Level.A
+        }
+    }
+}
+
+extension BadPersonality {
+    var isBadPerson: Bool {
+        get {
+            self.frequancyOfCursing == Level.A
+        }
+    }
+}
+
+struct AuditionManager {
+    let totalApplicantsList: [Person]
+    var passedApplicantsList: [Person] = []
+    
+    mutating func cast() {
+        guard !totalApplicantsList.isEmpty else { print("지원자가 없습니다."); return }
```

Comment:
지원자가 없는 경우도 고려하셨군요! 👍
비교적 간단한 코드이니 괜찮다고 생각하긴 합니다만, 스위프트에서 세미콜론은 잘 사용하지 않긴 합니다. 😅
```suggestion
        guard !totalApplicantsList.isEmpty else { 
            print("지원자가 없습니다.")
            return 
        }
```

이게 정답이라는 뜻은 아닙니다! 세미콜론 사용이 더 가독성이 좋다고 생각하시면 저는 사용해도 괜찮다 생각해요 😉